tests/hooks/test_topical_memory_injection.py

```
@@ -0,0 +1,166 @@
+        rc, cap = self._run(tmp_path, stdin, monkeypatch, capsys)
+        assert rc == 0
+        assert cap.out.strip() == ""
+
+    def test_silent_when_no_memories_dir(self, tmp_path, monkeypatch, capsys):
+        stdin = _stdin_for(tmp_path)
+        rc, cap = self._run(tmp_path, stdin, monkeypatch, capsys)
+        assert rc == 0
+        assert cap.out.strip() == ""
+
+    def test_bad_stdin_exits_zero(self, tmp_path, monkeypatch, capsys):
+        rc, cap = self._run(tmp_path, "{not json", monkeypatch, capsys)
```

Ruff (0.15.14)

[warning] Unpacked variable `cap` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)